mariza/styles.css

```
@@ -0,0 +1,176 @@
+* {
+  box-sizing: border-box;
+}
+.header {
+  background-color: #ff8c00;
+  display: inline-block;
+  color: #ffffff;
+  font-family: "Open Sans", sans-serif;
+  width: 100%;
+}
+h1 {
+  background-color: #ff8c00;
+  font-size: 30px;
+  font-style: regular;
+  font-weight: medium;
+  font-family: "Raleway", sans-serif;
+  color: white;
+}
+.hours {
+  display: inline-block;
```

Since you don't want these elements to appear side-by-side, it is better to make this element a `div` instead of a `span` and lose this `display` rule. See the simplified CodePen example: https://codepen.io/angeliquejw/pen/qBOBgvE?editors=1100

```suggestion
.hours {
```